### PR TITLE
manage subject

### PR DESCRIPTION
**Pull request by VuongLong**

### CODE REVIEW

**Comment by binhthanhnguyen** [comment]
ok

**Comment by thiennv57** [comment]
OK

File: app/views/admin/subjects/update.js.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$("#nav_subject_edit").html("");
+$("#subject-<%= @subject.id %>").html("<%= escape_javascript(render "subject",subject: @subject) %>");
```
Comment (ThuBM) [inline app/views/admin/subjects/update.js.erb]: indent: add space before subject:

**Comment by ThuBM** [comment]
OK

**Pull request merged**